src/redux/books/books.js

```diff
@@ -11,6 +15,23 @@ export const removeBook = (payload) => ({
   payload,
 });
 
+export const fetchData = () => (dispatch) => fetch(
+  'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hU7hRqGMmMkUTHelAE4I/books',
+)
+  .then((response) => response.json())
+  .then((data) => {
+    Object.keys(data).forEach((book) => {
+      dispatch({
+        type: ADD_BOOK,
+        payload: {
+          item_id: book,
+          ...data[book][0],
+        },
+      });
+    });
+  });
```

Comment:
- [ ] Kindly make sure that the book you created is stored in the API. Currently, if I created a book it does not store in the API thus the book disappears if I refresh the page. 😄 👍 

![Capture](https://user-images.githubusercontent.com/64173189/156694294-3c51ce56-fc31-4dd9-bba2-ce774207cca0.JPG)

![Capture](https://user-images.githubusercontent.com/64173189/156694335-8eddeb1c-cfdc-4646-a7ef-7db8dbdab536.JPG)

Reply:
Done